app/views/movies/index.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2><%= button_to 'Find Top Rated Movies', '/movies' %></h2>
+
+<%= form_with url: '/movies', method: 'get', local: true do |f| %>
+  <%= f.text_field :movie_title , placeholder: "Search by movie title"%>
+  <%= f.submit "Find Movies" %>
+<% end %>
+
+<% @titles.each do |title| %>
+  <div class="movie-column">
+    <ul class="movie-results"><%= title[:original_title] %></ul>
```

Comment:
Maybe we can turn movies into objects during a refactor so we don't have to use a hash here.